main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+        self._ring_combo.blockSignals(False)
+
+    def _composite_scalar_layer(self,
+                                rgba: np.ndarray,
+                                arr: np.ndarray,
+                                units: str,
+                                name: str,
+                                *,
+                                valid_mask: np.ndarray,
+                                mode: str,
+                                alpha: float,
+                                cmap_name: Any) -> np.ndarray:
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `cmap_name`

(ANN401)